webcomponents/tool/src/main/frontend/packages/sakai-rubrics/src/SakaiRubricsManager.js

```diff
@@ -21,6 +21,29 @@ export class SakaiRubricsManager extends RubricsElement {
     this.enablePdfExport = false;
   }
 
+  firstUpdated() {
+
+    const siteRubricsBlock = this.querySelector("#site_rubrics");
+
+    siteRubricsBlock.addEventListener("show.bs.collapse", () => {
```

Comment:
Do you think that event listeners should be removed at some point or is it fine to keep them around?

Reply:
We need them to change the arrow direction.

Reply:
I am thinking about it more from an performance perspective - cleaning up listeners when they are not needed anymore https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener. But not in every case cleanup of the listeners is needed, that's why I'm asking.

Reply:
They only get added once, in firstUpdated, so they should be ok. Once the modal elements are destroyed, the listeners will be automatically cleaned up. That should be okay, right?